frontend/src/lib/api/zaken.ts

```diff
@@ -8,10 +8,13 @@ export type PaginatedZaken = PaginatedResults<Zaak>;
  * Retrieve zaken using the configured ZRC service. For information over the query parameters accepted and the schema of
  * the response, look at the '/zaken/api/v1/zaken' list endpoint of Open Zaak.
  */
-export async function listZaken(
-  params?: URLSearchParams | Record<string, string>,
-) {
-  const response = await request("GET", "/zaken/", params);
+export async function listZaken(params?: Record<string, string>) {
```

Comment:
Even though we can refactor this quite easily, lets try to keep the naming of the function/endpoints consistent.

Reply:
Ah yes good point. Shall I leave the list and add a search function? since the list is anyway still supported :thinking:

Reply:
I think unused code can be removed :)